index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Amen and Blessing</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <nav id="navbar">
+      <a class="home-link" href="#projects">Home</a>
+      <a class="about-link" href="#">About</a>
+      <a class="contact-link" href="projects">Contact us</a>
+    </nav>
+
+    <section id="welcome-section">
+      <h1>Hey it's Amen and Blessing</h1>
+      <p>We are web developers</p>
+    </section>
+
+    <section class="flexbox-container" id="projects">
+      <div class="flexbox-card">
+        <h4 class="project-tile">This is our HTML/CSS projects</h4>
+        <img
+          class="flexbox-images"
+          src="image.jpg"
+          alt="web developer in office"
+        />
+        <a
+          href="https://github.com/Amen-Musingarimi/AmenAndBlessing-CodingProject"
+          target="_blank"
+          id="profile-link"
+          >Click Here</a
+        >
+      </div>
+      <div class="flexbox-card">
+        <h4 class="project-tile">This is where we learn web development</h4>
+        <img class="flexbox-images" src="computer.jpg" alt="computer" />
+        <a href="https://www.microverse.org/" target="_blank" id="profile-link"
+          >Click Here</a
+        >
+      </div>
+      <div class="flexbox-card">
+        <h4 class="project-tile">Here is our codeing chanllenges</h4>
+        <img class="flexbox-images" src="code.jpg" alt="code" />
+        <a
+          href="https://www.freecodecamp.org/"
+          target="_blank"
+          id="profile-link"
+          >Click Here</a
+        >
+      </div>
```

Comment:
I think it would be semantically correct to wrap the images in an unordered list. The divs used are not semantic.